enterprise-suite/tests/smoke_prometheus

```diff
@@ -104,9 +104,6 @@ prom_has_no_data 'kube_pod_info{es_workload=""}'
 prom_has_no_data '{__name__=~ "kube_.+", pod!="", es_workload=""}'
 # all container data have a workload label
 prom_has_no_data '{__name__=~"container_.+", es_workload=""}'
-# all targets should be reachable
-prom_has_data 'up{kubernetes_name != "es-test-service-with-only-endpoints"} == 1'
```

Comment:
I think you can leave the first one.  It's the second one that gets us into trouble...

Reply:
Changed my mind.  As you were.